tests/ut/test_tracer.cc

```diff
@@ -22,20 +24,47 @@ using namespace opentelemetry::trace;
 TEST_CASE("Test Tracer init", "Init test") {
     using Catch::Approx;
 
-    auto config = std::make_shared<TraceConfig>();
-    config->exporter = "stdout";
-    config->nodeID = 1;
-    initTelemetry(*config);
-    auto span = StartSpan("test");
-    REQUIRE(span->IsRecording());
+    SECTION("check stdout") {
+        auto config = std::make_shared<TraceConfig>();
+        config->exporter = "stdout";
+        config->nodeID = 1;
+        initTelemetry(*config);
+        auto span = StartSpan("test");
+        SetRootSpan(span);
+        REQUIRE(span->IsRecording());
+        AddEvent("sleep");
+        usleep(2000);
+        span->End();
+        CloseRootSpan();
+    }
+
+    SECTION("check jaeger") {
+        auto config = std::make_shared<TraceConfig>();
+        config->exporter = "jaeger";
+        config->jaegerURL = "http://localhost:14268/api/traces";
```

Comment:
would it be possible to add some comments about the URL? Because it is unclear whether a unit test is going to launch a local socket/server that listens on this port. Or, maybe, an external optional tracing server might be used if available.

Reply:
add some comment